### ISSUE TITLE
Unable to install rdhs or run from existing library

### ISSUE THREAD

**Issue by sumalibajaj** [OJWatson/rdhs#32]
```
Warning in strptime(x, fmt, tz = "GMT") :
  unknown timezone 'zone/tz/2018c.1.0/zoneinfo/Europe/London'
Error: package or namespace load failed for ‘rdhs’:
 .onAttach failed in attachNamespace() for 'rdhs', details:
  call: handle_api_response(updates)
  error: DHS API Request Failed [500] Error Type: dhs_internal_server_error
Error: loading failed
Execution halted
ERROR: loading failed
* removing ‘/Library/Frameworks/R.framework/Versions/3.4/Resources/library/rdhs’
* restoring previous ‘/Library/Frameworks/R.framework/Versions/3.4/Resources/library/rdhs’
Installation failed: Command failed (1)
```
![screen shot 2018-05-17 at 1 09 13 pm](https://user-images.githubusercontent.com/34308572/40176572-b55fe9ee-59d3-11e8-8173-0643d7e0ec3e.png)

**Comment by OJWatson** [comment]
Hey Sumali, 

Thanks for flagging this up, and glad you got it working. This error has thrown as the API will have been down when you tried to install the package. However, this should not be the default behaviour. Have added a catch for this so hopefully this doesnt happen again. 

The other thing that is surprising is the `Warning in strptime` bit, as i don't know how that has come about. If you don't mind could you please try running the following and copying the output you get:

```
Sys.timezone()
Sys.getenv("TZ")
```

(For me: Perhaps related to [this](https://github.com/tidyverse/lubridate/issues/615))

**Comment by sumalibajaj** [comment]
That sounds good!
Following is the output:
```
> Sys.timezone()
[1] NA
> Sys.getenv("TZ")
[1] ""
```
Does that help? Let me know if you would want me to do anything else?

Thanks!

**Comment by OJWatson** [comment]
okay cool this looks like a potential issue with mac os high sierra and certain R versions. Can your provide your session info (`sessionInfo()`)

**Comment by sumalibajaj** [comment]
Okay, sure:
```
> sessionInfo()
R version 3.4.2 (2017-09-28)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS High Sierra 10.13.3

Matrix products: default
BLAS: /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.4/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] grid      stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] rdhs_0.4.0          plyr_1.8.4          dplyr_0.7.4         dummies_1.5.6      
 [5] rstan_2.17.3        StanHeaders_2.17.2  ggplot2_2.2.1       foreign_0.8-70     
 [9] maptools_0.9-2      sp_1.2-7            survey_3.33-2       survival_2.42-3    
[13] Matrix_1.2-14       data.table_1.10.4-3

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.16     cellranger_1.1.0 bindr_0.1.1      compiler_3.4.2  
 [5] pillar_1.2.2     git2r_0.21.0     forcats_0.3.0    tools_3.4.2     
 [9] digest_0.6.15    jsonlite_1.5     memoise_1.1.0    tibble_1.4.2    
[13] gtable_0.2.0     lattice_0.20-35  pkgconfig_2.0.1  rlang_0.2.0     
[17] curl_3.2         yaml_2.1.19      haven_1.1.1.9000 bindrcpp_0.2.2  
[21] gridExtra_2.3    withr_2.1.2      storr_1.1.3      httr_1.3.1      
[25] knitr_1.20       rappdirs_0.3.1   devtools_1.13.5  stats4_3.4.2    
[29] glue_1.2.0       inline_0.3.14    R6_2.2.2         readxl_1.1.0    
[33] magrittr_1.5     scales_0.5.0     splines_3.4.2    assertthat_0.2.0
[37] colorspace_1.3-2 lazyeval_0.2.1   munsell_0.4.3   
```

**Comment by OJWatson** [comment]
Okay cool - this is I think a replicate issue of this then (https://stackoverflow.com/questions/47314121/r-error-unknown-timezone-with-as-posixct/47467017#47467017)

Issue seems to be that on macOS High Sierra with R version < 4.3.0 then this occurs where there is no timezones set by default. 

Solutions (any of the following should help with this, in roughly order of suitability):

1. Upgrade to R v.4.3.0
2. Set a timezone:
```
Sys.setenv(TZ = "Europe/London")
```
3. Place the above within your `.Rprofile` file and then it should be remembered with each new R session. Try using `usethis::edit_r_profile` to easily edit this file.

**Comment by sumalibajaj** [comment]
Great works now! Thank you very much.

**Comment by OJWatson** [comment]
Glad to hear it's worked. Am closing now (fix in #5685db5)
